### PR TITLE
media-sound/sndio: Add 1.9.0

### PR DESCRIPTION
**Pull request by Marinapig**
Signed-off-by: Henry Ford fordhenry2299@gmail.com

### CODE REVIEW

**Comment by scardracs** [comment]
Please drop https://github.com/gentoo/gentoo/pull/29280/commits/b74cf98559610c2d5582037ca651f28508fd9f54 as it will be a pain in ass during merge. Prefer git rebase master instead of git merge master

File: media-sound/sndio/sndio-1.9.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Copyright 2020-2022 Gentoo Authors
```
Comment (scardracs) [inline media-sound/sndio/sndio-1.9.0.ebuild:1]: 1999-2023

File: media-sound/sndio/sndio-1.9.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (scardracs) [inline media-sound/sndio/sndio-1.9.0.ebuild:4]: EAPI=8

File: media-sound/sndio/sndio-1.9.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+DESCRIPTION="small audio and MIDI framework part of the OpenBSD project"
+HOMEPAGE="http://www.sndio.org/"
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://caoua.org/git/sndio"
+	EGIT_MIN_CLONE_TYPE="single+tags"
+else
+	SRC_URI="http://www.sndio.org/${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~ppc ~ppc64 ~riscv ~sparc ~x86"
+fi
```
Comment (scardracs) [inline media-sound/sndio/sndio-1.9.0.ebuild:17]: As this package has live ebuild too please bump it with changes I'm listing

**Review by scardracs — CHANGES_REQUESTED** [review]

**Comment by Marinapig** [comment]
Got rid of that merge and integrated your changes

File: media-sound/sndio/sndio-1.9.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2022 Gentoo Authors
```
Comment (scardracs) [inline media-sound/sndio/sndio-1.9.0.ebuild:1]: Sorry, I meant 2020-2023

File: media-sound/sndio/sndio-9999.ebuild [hunk 5]
```diff
@@ -1,7 +1,7 @@
-# Copyright 2020-2022 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
```
Comment (scardracs) [inline media-sound/sndio/sndio-9999.ebuild:1]: Same

**Comment by scardracs** [comment]
LGTM, anyway you have to wait for a dev for merge (I'm not one).

File: media-sound/sndio/sndio-1.9.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+multilib_src_configure() {
+	tc-export CC
+
+	./configure \
+		--prefix="${EPREFIX}"/usr \
+		--libdir="${EPREFIX}"/usr/$(get_libdir) \
+		--privsep-user=sndiod \
+		--with-libbsd \
+		$(use_enable alsa) \
+	|| die "Configure failed"
```
Comment (juippis) [inline media-sound/sndio/sndio-1.9.0.ebuild:48]: You can use `econf` here. Also an array form is better.

**Comment by lanodan** [comment]
>You can use `econf` here.

Nope.
It's not an autotools-style ./configure so some of the arguments econf uses do not work.

> Also an array form is better.

:shrug:

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**